docs/product/components/tables.html

```diff
@@ -757,7 +757,7 @@
                 {% for item in tables.data-attributes %}
                     <tr>
                         <th scope="row"><code class="stacks-code">{{ item.attribute }}</code></th>
-                        <td><code class="stacks-code">{{ item.applies }}</td>
+                        <td><code class="stacks-code">{{ item.applies }}</code></td>
```

Comment:
[Comment related to the docs snippet above]
Do you think that this part of the documentation (see screenshot) should be updated too with a the new markup button to avoid confusion?
At the moment it seems to use the old markup
<img width="820" alt="Screenshot 2023-05-09 at 17 12 55" src="https://github.com/StackExchange/Stacks/assets/6231616/821b2ea7-2908-47fc-ac9b-896cad4355bf">

Reply:
On second look, I take it back. The above example comes before the "JavaScript sorting" section and is static. The purpose of this example is to show a _sorted_ table and, IMO, shows that Stacks can provide the appearance of sorting without the ability for a user to interact with that sorting. I'm going to leave this example as-is but let me know if you have a strong opinion and think it should be updated.

Reply:
@dancormier One thing that confuses me about that example is that it gives the impression to the user the table header can be interacted with: not just because of the carets icons but also because of the cursor becoming a `pointer` when hovering on top of it. 
I believe that there is not a real usecase for what we showcase there and it's going to trick people into forgetting to add the buttons. A "sortable table" can always be interacted with and should use buttons. A "sorted table" should not have the double carets symbols in the column which are not sorted and should not show a pointer as cursor (not even sure if that's a concrete use case).